tests/gql_behave/tests/memgraph_V1/features/list_operations.feature

```diff
@@ -291,3 +291,63 @@ Feature: List operators
 #        Then the result should be:
 #            | years                 |
 #            | [2021,2003,2003,1999] |
+
+     Scenario: Multiple entries with list pattern comprehension
+        Given graph "graph_keanu"
+        When executing query:
+            """
+            MATCH (n:Person)
+            RETURN n.name, [(n)-->(b:Movie) WHERE b.title CONTAINS 'Matrix' | b.released] AS years
+            """
+        Then an error should be raised
+#        Then the result should be:
+#            | n.name               | years                 |
+#            | "Keanu Reeves"       | [2021,2003,2003,1999] |
+#            | "Carrie-Anne Moss"   | [2003,1999]           |
+#            | "Laurence Fishburne" | [1999]                |
```

Comment:
What is the plan for commented-out code?

Reply:
I will uncomment one of expected result when execution of RollUpApply will be implemented.
Later will be uncommented others when we will add implementation for more cases.

Reply:
I would remove them since (as I understand) it will not be implemented in the next PR. And you can document somewhere (on Notion) how results should look like when/if someone in the future adds support for that feature.